panel/models/anywidget_component.ts

```diff
@@ -0,0 +1,83 @@
+import type * as p from "@bokehjs/core/properties"
+
+import {ReactComponent, ReactComponentView} from "./react_component"
+
+const panel_adapter = `
+class PanelModel {
+
+  constructor(model) {
+    this.model = model;
+  }
+
+  get(name) {
+    let value
+    if (name in this.model.attributes) {
+      value = this.model[name]
+    } else {
+      value = this.model.data[name]
+    }
+    if (value instanceof ArrayBuffer) {
+      value = new Uint8Array(value)
+    }
+    return value
+  }
+
+  set(name, value) {
```

Comment:
Would there not be a use case for setting model values as well. Like width or height?